src/Resources/config/services.yml

```diff
@@ -9,7 +9,7 @@ services:
         arguments:
             - "@service_container"
         tags:
-            - { name: kernel.event_listener, event: kernel.controller, method: onKernelController, priority: 254 }
+            - { name: kernel.event_listener, event: kernel.controller, method: onKernelRequest, priority: 31 }
```

Comment:
Priority 32 is the routing listener, priority 30 is the `InitializeSystemListener`. The container scope must be set between them.